docs/sce_file_contents.md

```diff
@@ -133,3 +133,65 @@ Finally, additional metadata for the CITE-seq data processing can be found in th
+Column names are again cell barcode sequences and row names the HTO ids for all assayed HTOs. 
+
+The following additional per-cell data columns for the cellhash data can be found in the main `colData` data frame (accessed with `colData(sce)` [as above](#cell-metrics)).
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `altexps_cellhash_sum  `    | UMI count for cellhash HTOs                       |
+| `altexps_cellhash_detected` | Number of HTOs detected per cell (HTO count > 0 ) |
+| `altexps_cellhash_percent`  | Percent of `total` UMI count from HTO reads       |
+
+Metrics for each of the HTOs assayed can be found as a `DataFrame` stored as `rowData` within the alternative experiment:
+
+```r
+hto_info <- rowData(altExp(sce, "cellhash"))
+```
+
+This data frame contains the following columns with statistics for each HTO:
+
+| Column name | Contents                                                       |
+| ----------- | -------------------------------------------------------------- |
+| `mean`      | Mean HTO count across all cells/droplets                       |
+| `detected`  | Number of cells in which the HTO was detected (HTO count > 0 ) |
+| `sample_id` | Sample ID for this library that corresponds to the HTO (only present in `_filtered.rds` files) |
+
+Note that in the unfiltered `SingleCellExperiment` objects, this may include hashtag oligos that do not correspond to any included sample, but were part of the reference set used for mapping.
+
+### Demultiplexing results
+
+Demultiplexing results are included only in the `_filtered.rds` files. 
+The demultiplexing methods applied for these files are as described in the {ref}`multiplex data processing section <processing_information:Multiplexed libraries>`.
+
+Demultiplexing analysis adds the following additional fields to the `colData(sce)` data frame:
+
+| Column name | Contents                                                       |
+| ----------- | -------------------------------------------------------------- |
+| `hashedDrops_sampleid`  | Most likely sample as called be `DropletUtils::hashedDrops` 
+| `HTODemux_sampleid`  | Most likely sample as called be `Seurat::HTODemux`
+| `vireo_sampleid`  | Most likely sample as called by `vireo` (genetic demultiplexing)        
+
+### Additional demultiplexing statistics
+
+Each demultiplexing method generates additional statistics specific to the method that you may wish to access, including probabilities, alternative calls, and potential doublet information.
+These statistics are included in the `SingleCellExperiment` component `DataFrames` with prefixes corresponding to the method from which they were derived.
```

Comment:
This sentence is confusing to me. I don't think you need it since you clearly define where they are located in the next sentences. 
```suggestion
```